scripts/trigger-mobile-metrics.py

```diff
@@ -75,6 +75,7 @@ def Main():
 
   # These jobs need to be refactored into workflows.
   if publishResults:
+    TriggerJob(token, commit, "android-navigation-benchmark")
```

Comment:
cc @kmadsen

Reply:
do you want to remove the TriggerJob's for `android-navigation-code-coverage` and `android-navigation-binary-size`. or are they working?

should keep the dry run ones though.

Reply:
yeah, wanted to sync. Think this's a good place to disable these jobs 👍

Reply:
yeah just comment out the publishResults lines
```
    TriggerJob(token, commit, "android-navigation-code-coverage")
    TriggerJob(token, commit, "android-navigation-binary-size")
```

but keep the triggers in the else block, so we can continue to test them